cmd/fleetctl/gitops_test.go

```diff
@@ -2652,7 +2652,7 @@ func setupFullGitOpsPremiumServer(t *testing.T) (*mock.Store, **fleet.AppConfig,
 	}
 	ds.TeamByFilenameFunc = func(ctx context.Context, filename string) (*fleet.Team, error) {
 		for _, tm := range savedTeams {
-			if *(*tm).Filename == filename {
+			if (*tm).Filename != nil && *(*tm).Filename == filename {
```

Comment:
This was failing in my test where I added a team outside of GitOps, adding nil check for safety.